### PR TITLE
Support Bootstrap 4

### PR DESCRIPTION
**Pull request by justin808**

### CODE REVIEW

File: bootstrap-no-customizations.config.js [hunk 1]
```diff
@@ -3,62 +3,61 @@ module.exports = {
   styleLoader: 'style-loader!css-loader!sass-loader',
 
   scripts: {
-    'transition': true,
+    'util': true,
```
Comment (justin808) [inline bootstrap-no-customizations.config.js]: We'll need different manifest files for Bootstrap 3 and Bootstrap 4

File: bootstrap-no-customizations.config.js [hunk 2]
```diff
@@ -3,62 +3,61 @@ module.exports = {
   styleLoader: 'style-loader!css-loader!sass-loader',
```
Comment (justin808) [inline bootstrap-no-customizations.config.js]: How about if we simply have a property:

```
 bootstrapVersion: 3,
```

or 

```
 bootstrapVersion: 4,
```

**Comment by justin808** [comment]
@alexfedoseev Please double check we're putting files in the right order.



Also, please add some comments on the difference between the "main" sass file as well as the "bootstrap-customization file".

While you could put all post-bootstrap loading customizations in the bootstrap-customization file, you separate out that to a second sass file which is your own app's sass files. Your app's sass files will be able to any sass variables defined by bootstrap.

CC: @robwise

**Comment by maxbates** [comment]
Any idea on when this will be ready to go?

**Comment by justin808** [comment]
@maxbates It's just about ready. We could definitely use some help with testing and documentation.

http://forum.shakacode.com/t/open-source-help-wanted/416

**Comment by justin808** [comment]
Closing as we're about to put in the PR that will take this loader to the next level. 

See it in action here: https://github.com/shakacode/react-webpack-rails-tutorial/pull/175

**Pull request closed without merging**